client/src/components/questionsAnswers/AnswerReport.jsx

```diff
@@ -23,16 +22,14 @@ class AnswerReport extends React.Component {
       });
     }
   }
-
   render() {
     return (
-      <div>
+      <div onClick={() => {interact('div', 'AnswerReportButton')}}>
```

Comment:
I haven't tried declaring a function inside of a click event like this before. I'll have to give it a try.